src/node.cc

```diff
@@ -3485,6 +3489,9 @@ static void PrintHelp() {
          "  --throw-deprecation      throw an exception on deprecations\n"
          "  --no-warnings            silence all process warnings\n"
          "  --trace-warnings         show stack traces on process warnings\n"
+         "  --redirect-warnings=path\n"
+         "                        write process warnings to the given file\n"
+         "                        path rather than to stderr\n"
```

Comment:
If it helps shorten this, I think you can just say `--redirect-warnings=path` and `write process warnings to path rather than to stderr`.